connector_jira/components/backend_adapter.py

```diff
@@ -96,12 +96,12 @@ def handle_user_api_errors(self):
         except requests.exceptions.ConnectionError as err:
             _logger.exception("Jira ConnectionError")
             message = _("Error during connection with Jira: %s") % (err,)
-            raise exceptions.UserError(message)
+            raise exceptions.UserError(message) from err
```

Comment:
Why do we need to add `from err` to `raise` statements?

Reply:
This one for distinguish from errors in exception chain, and to pass pre-commit run
[exception document](https://docs.python.org/3/tutorial/errors.html#exception-chaining)